Update REAME with the Web Interface introduction (#431)
meilisearch/MeiliSearch: README.md

}
```

#### With the Web Interface

> MeiliSearch provides a simple web interface containing a search bar so that you can quickly experience the instant search with your own documents.

Reviewer: ```suggestion
MeiliSearch provides a simple web interface containing a search bar in order to quickly test the instant search experience with a given set of documents. 
```

> This web interface is available on the port `7700` in your browser.

Reviewer 2: ```suggestion
This web interface is available in your browser at the root of the server, the default URL is [http://127.0.0.1:7700](http://127.0.0.1:7700).
```
